lib/utils/udpsender.js

```diff
@@ -0,0 +1,20 @@
+require('./KalturaConfig');
+
+var PORT = KalturaConfig.config.udpsender.PORT;

```

Comment:
I think that lower case convention is more suitable both for the variable name and the config item name.